packages/rum/src/domain/segmentCollection/segment.ts

```diff
@@ -9,27 +9,30 @@ let nextId = 0
 export class Segment {
   public isFlushed = false
   public flushReason?: string
+  public readonly meta: SegmentMeta
```

Comment:
Does `meta` means `metadata`?

Reply:
Yes

Reply:
Then I would find it less confusing using the naming `metadata`

Reply:
Sure